src_plugins/eq/ppx_deriving_eq.ml

```diff
@@ -94,11 +94,11 @@ and expr_of_typ quoter typ =
           | Some a, Some b -> [%e expr_of_typ typ] a b
           | _ -> false]
       | true, ([%type: ([%t? ok_t], [%t? err_t]) result] |
-               [%type: ([%t? ok_t], [%t? err_t]) Result.result]) ->
+               [%type: ([%t? ok_t], [%t? err_t]) result]) ->
```

Comment:
The two cases are now the same. Same in a bunch of other plugins.

But I wonder what implications this change has because this is a fully syntactic pattern. Code which previously derived explicitly for `Result.result` would no longer match here and fail to derive, no?

Reply:
that's fair. I didn't catch this as it was mostly find / replace. I think it might probably be warranted to keep this case here, which is just matching on an ident `Result.result`. Since that doesn't re-introduce back the result dependency I think it should be reverted.